cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -92,9 +104,20 @@ func (t *TransferPVCOptions) Complete(c *cobra.Command, args []string) error {
 		return err
 	}
 
```

Comment:
Can we add a block to default the destination PVC name to the source PVC name if empty?